docs/troubleshoot/cli/cli-use-curl-to-troubleshoot.md

```diff
@@ -0,0 +1,101 @@
+# Using cURL to troubleshoot Zowe CLI
+
+When a REST API call fails on Zowe CLI, you can use cURL to troubleshoot.
```

Comment:
Users may be unsure whether they have cURL installed. Should we add something like the following to the introductory paragraph?
> cURL is installed by default on Windows 10 versions 1803 and later, macOS, and most Linux distros. If the `curl` command is missing from your system, you can download it [here](https://curl.se/download.html).

Reply:
Instead of using version numbers, should we try to use the date format for the win10 update?
That way we can change the wording to "Windows 10  after the April 2018 update" ?

Reply:
I think we should pick whichever is easier for users to check. Personally I like the version number because I can go to System Settings and see whether I have that version number or not. I don't remember what date I last updated Windows. But maybe I'm weird in this regard 😀

Reply:
I think using the v number is better in this case